src/components/BookItem.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const BookItem = ({
+  id, title, author, categories,
+}) => (
+  <li key={id}>
+    <p>{title}</p>
+    <p>{author}</p>
+    <p>{categories}</p>
+
+  </li>
+);
```

Comment:
- [ ] Your bookItem component should have a remove book button to comply with the project requirements, it does not need to work yet, just to be there, please add it 👍

Reply:
Thank you @e71az I added a remove button as you can see in the image below:
![Button](https://user-images.githubusercontent.com/60292761/135031182-99ccd02b-092a-4e4a-8010-99261e676651.PNG)